sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_queues_async.py

```diff
@@ -329,11 +329,11 @@ async def test_async_mgmt_queue_update_invalid(self, servicebus_namespace_connec
         queue_description = await mgmt_service.create_queue(queue_name)
         try:
             # handle a null update properly.
-            with pytest.raises(AttributeError):
+            with pytest.raises(TypeError):
                 await mgmt_service.update_queue(None)
 
             # handle an invalid type update properly.
-            with pytest.raises(AttributeError):
+            with pytest.raises(TypeError):
```

Comment:
is it a breaking change -- raising TypeError instead of AttributeError?

Reply:
As discussed in the chat - we can probably call this a bugfix

Reply:
so I guess the change is worth a line in the changelog?
something like:
```
Fixed a bug that `update_*` methods on `ServiceBusManagementClient` should raise `TypeError` when given value of invalid types instead of `AttributeError`.
```

Reply:
Added! :)